File: gestion-voyage/src/Entités/Voyage.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entités;

/**
 *
 * @author ASUS
 */
public class Voyage {
    private int id, prix;
    private String nom, destination, description;
    

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getDestination() {
        return destination;
    }

    public void setDestination(String destination) {
        this.destination = destination;
    }
     public String getDescription() {
        return description;
    }
      public void setDescription(String description) {
        this.description = description;
    }
        public int getPrix() {
        return prix;
    }
      public void setPrix(int prix) {
        this.prix = prix;
    }
      
    public Voyage() {
    }

    public Voyage(int id, String nom, String destination, String description, int prix) {
        this.id = id;
        this.nom = nom;
        this.destination = destination;
        this.description = description;
        this.prix = prix;
    }
        public Voyage(String nom, String destination, String description, int prix) {
       
        this.nom = nom;
        this.destination = destination;
        this.description = description;
        this.prix = prix;
    }

    @Override
    public String toString() {
        return "Voyage{" + "id=" + id + ", nom=" + nom + ", destination=" + destination +  ", description=" + description + ", prix=" + prix +'}';
    }
    
    
}

File: Desktop/gestionevenement/src/Entity/Voyage.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Voyage
 *
 * @ORM\Table(name="voyage")
 * @ORM\Entity
 */
class Voyage
{
    /**
     * @var int
     *
     * @ORM\Column(name="Id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="Nom", type="string", length=150, nullable=false)
     */
    private $nom;

    /**
     * @var string
     *
     * @ORM\Column(name="Destination", type="string", length=30, nullable=false)
     */
    private $destination;

    /**
     * @var string
     *
     * @ORM\Column(name="Description", type="string", length=2000, nullable=false)
     */
    private $description;

    /**
     * @var int
     *
     * @ORM\Column(name="Prix", type="integer", nullable=false)
     */
    private $prix;

    /**
     * @var string
     *
     * @ORM\Column(name="Image", type="string", length=150, nullable=false)
     */
    private $image;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNom(): ?string
    {
        return $this->nom;
    }

    public function setNom(string $nom): self
    {
        $this->nom = $nom;

        return $this;
    }

    public function getDestination(): ?string
    {
        return $this->destination;
    }

    public function setDestination(string $destination): self
    {
        $this->destination = $destination;

        return $this;
    }

    public function getDescription(): ?string
    {
        return $this->description;
    }

    public function setDescription(string $description): self
    {
        $this->description = $description;

        return $this;
    }

    public function getPrix(): ?int
    {
        return $this->prix;
    }

    public function setPrix(int $prix): self
    {
        $this->prix = $prix;

        return $this;
    }

    public function getImage(): ?string
    {
        return $this->image;
    }

    public function setImage(string $image): self
    {
        $this->image = $image;

        return $this;
    }


}
